test/src/tests-store.ts

```diff
@@ -1814,4 +1856,29 @@ describe('Store', () => {
             });
         });
     });
+
+    describe('integration tests', () => {
+        describe('#firstSeenTime', () => {
+            it('should be null for users in storage without an lst value', done => {
+                const cookies = JSON.stringify({
+                    gs: {
+                        sid: 'lst Test',
+                        les: new Date().getTime(),
+                    },
+                    previous: {},
+                    cu: 'current',
```

Comment:
This applies to all the tests you are changing.  (And future updates). 

I'd say instead of giving the sample MPIDs here `previous` and `current`, we shold call it `PREVIOUS_MPID`/`previous_mpid` and `CURRENT_MPID`/`current_mpid`. This maes it more clear to someone who is reading this code that the key is an `MPID`.